rules/network/command_and_control_nat_traversal_port_activity.toml

```diff
@@ -20,7 +20,7 @@ false_positives = [
     port in the range by coincidence. This is uncommon but such servers can be excluded.
     """,
 ]
-index = ["filebeat-*"]
+index = ["filebeat-*", "packetbeat-*"]
```

Comment:
wondering if we could add `auditbeat-*` here too--it has some kprobe-based flow data coming from the `system` module `socket` dataset:

https://github.com/elastic/beats/blob/master/x-pack/auditbeat/tests/system/test_system_socket.py#L260

Reply:
Created #35 @andrewstucki -- we'll come back to this in a separate PR

Reply:
sweet, and didn't see that in the linked issue someone already beat me to the suggestion